src/service/service.ts

```diff
@@ -41,6 +43,7 @@ export default function(
       if (webviewPanel.visible) {
         webviewPanel.webview.postMessage(webviewMessage);
       } else {
+        treeDataProvider.updateWhiteboardState(true);
```

Comment:
This could potentially lead to multiple updates to the tree view, and we can avoid that by maintaining state in the tree provider.

Reply:
Good point! I’ll make that change.